packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorMenuItemTests.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Linq.Expressions;
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleAnchorMenuItem"/>.
+/// </summary>
+public class NimbleAnchorMenuItemTests
```

Comment:
Can we make an AnchorBase Xunit theory test or w/e it's called to make sure we are exercising all the shared link properties in one spot?